src/bundle/Resources/public/js/scripts/fieldType/ezdate.js

```diff
@@ -85,30 +83,41 @@
         const sourceInput = field.querySelector(SELECTOR_INPUT);
         const flatPickrInput = field.querySelector(SELECTOR_FLATPICKR_INPUT);
         const btnClear = field.querySelector('.ez-data-source__btn--clear-input');
+        let flatpickrInstance = null;
         let defaultDate;
 
         if (sourceInput.value) {
             defaultDate = new Date(sourceInput.value * 1000);
         }
 
-        btnClear.addEventListener('click', (event) => {
-            event.preventDefault();
+        btnClear.addEventListener(
+            'click',
+            (event) => {
```

Comment:
Maybe it would be good to define callback in some variable?

Reply:
I didn't want to change to much at first, but in last commit I refactored it. :)